site/source/user-manual/backups/backup-create.rst

```diff
@@ -1,41 +1,21 @@
 .. _backup-create:
 
-=============
-Backup Create
-=============
+======
+Backup
+======
 
 Backing up your Embassy is easy and secure. Backups are encrypted with your master password.
 
-Backup Using LAN Shared Folder
-------------------------------
-
 .. tip:: This is the recommended approach for creating backups.
 
 #. Follow instructions for creating a :ref:`LAN Shared Folder<cifs-setup>` on your laptop/desktop.
 
-#. Go to *Embassy > Create Backup*.
+#. Go to "Embassy" > "Create Backup".
 
     .. figure:: /_static/images/config/embassy_backup.png
         :width: 60%
 
-#. Next, click on "New Shared Folder".
-
-    .. figure:: /_static/images/config/embassy_backup0.png
-        :width: 60%
-
-#. Fill in the following fields:
-
-    * Hostname - This is the hostname of the machine that your shared folder is located on
-    * Path - This is the directory path to the shared folder. If you configured Samba yourself (Linux), this is the "comment" (name of the share in your samba config file) and not the path.
-    * Username - This is the user on the remote machine that you used to create the shared directory
-    * Password - This is your user (from above) password
-
-    .. figure:: /_static/images/config/embassy_backup1.png
-        :width: 60%
-
-#. Click "Save".
-
-#. You will see a freshly created "cloud" backup location available, click it, then click "Create Backup".
+#. You will see your previously created "cloud" (LAN Shared Folder) backup location (or :ref:`Secondary Drive<physical-drive>`) available.  Click it, then click "Create Backup".
```

Comment:
I don't like introducing this idea of "cloud" backups to describe LAN Shared Folders. We don't use that anywhere else, and I think it's inaccurate, considering we will soon be offering cloud backups